project-reports/2023/2023-Q4-Hyperledger-Besu.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: 2023 Q4 Hyperledger Besu
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Hyperledger Besu continues its momentum for usage on public and private Ethereum and Ethereum-like networks. We now have more accurate views of Besu's Mainnet usage ([here](https://etherclients.com/) and [here](https://www.ethernets.io/?synced=yes)). Besu is also seeing adoption on Layer 2 networks like Linea and the OP Stack for Optimism. We have one new Maintainer from Kaleido and hope to add more from Web3Labs. This quarter we saw improvements to Tracing / EVM performance and a new transaction pool implementation shipped. The client saw a number of bug fixes as a priority. The modularity work did not progress as much as intended...  Discord activity remains heavy, with a relatively even split between public and private network questions and suggestions. This quarter we added an enterprise-focused discord channel.  
+
+
+# Questions/Issues for the TOC
+
+How is the TOC planning to allocate resources for developer outreach and education that is not unduly burdensome on maintainers? 
```

Comment:
Can you provide more information on this question?  Which resources are you specifically looking to allocate? The TOC does not typically get involved with developer outreach and education. This has typically in the past been a collaboration between the maintainers and the Hyperledger Foundation staff members. Is there something specific that you think falls to the TOC?

Reply:
Maybe my TOC education has yet to begun for 2024 :) 

Hyperledger has been more active in providing these from the community. The goal is to continue providing these resources without making Maintainers blockers to releasing new content, tutorials, and workshops.